src/operator/fully_connected-inl.h

```diff
@@ -108,9 +110,12 @@ class FullyConnectedOp : public Operator {
           Shape2(oshape[0], oshape.ProdShape(1, oshape.ndim())), s);
     }
 
-    // Legacy approach shown here for comparison:
-    //   out = dot(data, wmat.T());
+#if (MSHADOW_USE_CBLAS == 0)
+    // Legacy approach for amalgamation build w/out cblas
+    out = dot(data, wmat.T());
```

Comment:
I'll merge this for now but eventually we should add a non-blas version into linalg_gemm